src/components/Table/components/TableHead/index.js

```diff
@@ -79,6 +90,11 @@ TableHead.propTypes = {
    * Enables/disables sticky columns on mobile
    */
   rowHeaders: PropTypes.bool,
+  /**
+   * [PRIVATE] Adds condensed styles the table according to the table props.
+   * Handled internally.
+   */
+  condensed: PropTypes.bool,
   /**
    * [PRIVATE] sortBy handler
```

Comment:
Hm, I see that this format has already been used elsewhere 🤔 Which one do you prefer? Would you mind updating these to the `@private` syntax as well?

Reply:
Done! :)